pkg/handler/handler.go

```diff
@@ -268,6 +268,8 @@ func (h handler) provision(w http.ResponseWriter, r *http.Request, params map[st
 		switch err {
 		case broker.ErrorDuplicate:
 			writeResponse(w, http.StatusConflict, broker.ProvisionResponse{})
+		case broker.ErrorProvisionInProgress:
+			writeResponse(w, http.StatusAccepted, broker.ProvisionResponse{})
```

Comment:
I think that we need to return the job token, like: `ProvisionResponse{Operation: token}`